### ISSUE TITLE
OS 10.11

### ISSUE THREAD

**Issue by ifightpolarbears** [aricallen/genius-scheduler#1]
roster genius stopped working which is how I've found your tool. however your tool doesn't work either. i get no feedback from the bookmark at all similar to roster genius. I'm hoping its a setting i can't find that just needs to be removed or changed. but a heads up, this doesn't work on 10.11

**Comment by aricallen** [comment]
I appreciate the feedback. Can you tell me what browser your using? I haven't updated to 10.11 yet so I'm unable to test but I might be able to help narrow down what the problem could be.

**Comment by ifightpolarbears** [comment]
Safari. I wonder if it has something to do with running beta. It's also disabled on on ios9. 

Cheers.

**Comment by aricallen** [comment]
Sorry I can't investigate this on my own yet... I don't feel comfortable updating to 10.11 just yet... I'll be sure to fix this as soon as I can. Stay tuned.

**Comment by ifightpolarbears** [comment]
No worries! 
Thanks for your time! 
Just wanted to let you know. 

Cheers.

**Comment by djmx87** [comment]
Hey there! iOS 9 just came out and it killed genius scheduler. I have manually fixed the code so it matches my link on my own website. But i'm not sure what to fix so it works on iOS 9. Can you please fix that? I'm assuming 10.11 will also have issues with it when it comes out. Please please help! I Literally just signed up so I can speak with you!

**Comment by aricallen** [comment]
Hello, I'm glad you've found use from my script. I know iOS 9 rendered it broken... I'm going to try to get around to fixing it asap but I've been very busy. Stay tuned.

**Comment by jmarich** [comment]
Hey there, I was wondering if you have had the chance to look at this?

I tried to figure out why it doesn't work but i can't really figure out why its not working anymore. I am assuming its the bookmarklet itself that is not sending the information to the scripts.

Any guidance would be greatly appreciated!

**Comment by djmx87** [comment]
Apple updated the site and the script doesn't work any longer. Before you had to change the JavaScript link to your own site link and import the files. Now that doesn't work. 

Sent from my iPhone

On Mon, May 2, 2016 at 1:19 PM -0700, "Juan Marich" notifications@github.com wrote:

Hey there, I was wondering if you have had the chance to look at this?

I tried to figure out why it doesn't work but i can't really figure out why its not working anymore. I am assuming its the bookmarklet itself that is not sending the information to the scripts.

Any guidance would be greatly appreciated!

**Comment by jmarich** [comment]
I did notice the issue happened before the site was changed, i assumed the script wouldn't work anymore on top of it all since the site is different, but the script is not even invoked from the very beginning, so i assumed that a good starting point would be to make sure the bookmarklet is working...

**Comment by djmx87** [comment]
That's correct. And the script it re-directs it to your personal site once everything is uploaded. 

Sent from my iPhone

On Mon, May 2, 2016 at 1:25 PM -0700, "Juan Marich" notifications@github.com wrote:

I did notice the issue happened before the site was changed, i assumed the script wouldn't work anymore on top of it all since the site is different, but the script is not even invoked from the very beginning, so i assumed that a good starting point would be to make sure the bookmarklet is working...

**Comment by jmarich** [comment]
i have had the files hosted on my server, and properly linked to the bookmarklet, but nothing at all happens, that is what has me stumped, and i have no idea where to even start looking

**Comment by aricallen** [comment]
Hi folks, I'm sorry I haven't been able to update this script. I no longer have access to the MyPage data in order to test so I couldn't really test. If someone emails me the page source of their schedule I may be able to come up with a patch but I won't be able to test on the live site.
